### PR TITLE
docs(codepush): add apply fix script

### PR DESCRIPTION
**Pull request by brainbicycle**
This PR resolves [] 

### Description

Adds an apply_fix script and updates the docs based on @damassi suggestion!

apply-fix-example

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- update hotfix docs and add apply fix script - Brian 

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (update hotfix docs and add apply fix script - Brian)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e1b29bcc8b65a70c42cf92daf7ca058ab8190856
</p>

**Review by gkartalis — DISMISSED** [review]
the comments are really helpful for understanding the script! 👏 🚢

File: docs/deploy_a_hotfix.md [hunk 1]
```diff
@@ -28,19 +28,14 @@ We will want to checkout the tag `ios-7.2.0-2022.02.03.14-submission`:
 
 `git checkout 7.2.0-hotfix`
 
-## Get the base native code hash of the release
+## Apply your fix using the script
 
-Run the script `./scripts/codepush/calculate-native-hash.sh`
-Note the output hash in the last line of the output. You will use this to determine if native code changed.
+Run the script `./scripts/codepush/apply-fix.sh <commit-hash>`
```
Comment (damassi) [inline docs/deploy_a_hotfix.md:33]: This is a bit unclear... What is the commit hash? Or is it the sort of thing where on the first run you don't provide one, and that's sufficient? _OR_ is that the commit hash of the change that was just made to fix the bug? 

If the latter, can we make that super clear? Like 

```
### create a branch for your hotfix 
... 

#### make changes and commit 
- `git commit -am "some fix"
- copy the hash of the commit from above: `git log --oneline | some bash

#### apply yr fix
- run the script: apply-fix.sh <hash-from-above>
...
``` 

That kind of thing

**Review by damassi — APPROVED** [review]
One minor docs elaboration request, but otherwise, real nice 👍

File: docs/deploy_a_hotfix.md (hunk 1 above)
Comment (brainbicycle) [reply docs/deploy_a_hotfix.md:33]: typically it is not the release captain doing the fix themselves and they are pulling it in from another persons branch but I can update the doc to make it more clear.

Comment (brainbicycle) [reply docs/deploy_a_hotfix.md:36]: updated! gonna merge but happy to update anything unclear

File: docs/deploy_a_hotfix.md [hunk 2]
```diff
@@ -22,25 +22,23 @@ We will want to checkout the tag `ios-7.2.0-2022.02.03.14-submission`:
...
 
 `git checkout 7.2.0-hotfix`
 
-## Get the base native code hash of the release
+## Apply the fix(es) using the script
 
-Run the script `./scripts/codepush/calculate-native-hash.sh`
-Note the output hash in the last line of the output. You will use this to determine if native code changed.
+Get the commit hash for the bug fix you want to release in the hotfix. If it was merged into main you want the hash of the merge commit.
+You can find it in the github ui or by checking out main after the merge and running `git log --oneline`.
```
Comment (damassi) [inline docs/deploy_a_hotfix.md:34]: Perfect

**Pull request merged**